automated install/basic-install.sh

```diff
@@ -2346,7 +2334,7 @@ make_temporary_log() {
 copy_to_install_log() {
     # Copy the contents of file descriptor 3 into the install log
     # Since we use color codes such as '\e[1;33m', they should be removed
-    sed 's/\[[0-9;]\{1,5\}m//g' < /proc/$$/fd/3 > "${installLogLoc}"
+    sed 's/\[[0-9;]\{1,5\}m//g' < /proc/$$/fd/3 > "${installLogLoc}"
```

Comment:
I cleaned the commit history but I have no idea how to add the above `` sign, as it is no UTF-8? Any ideas how to solve this?

Reply:
It's hex `1B` which is for `ESC`, I believe. https://en.wikipedia.org/wiki/Escape_character

![image](https://user-images.githubusercontent.com/1998970/124487987-3e558100-dda7-11eb-99b8-489f382936fc.png)

Not sure how one types it, though. How did you manage to alter the line?

May be easier just to start fresh perhaps?

Reply:
This is the result of copy-pasting it into a suggestion, which might work - but I think it's safer to start fresh with something that hasn't touched the line

```suggestion
    sed 's/\[[0-9;]\{1,5\}m//g' < /proc/$$/fd/3 > "${installLogLoc}"
```

Reply:
Okay, I re-added the changes now to a freshly downloaded script.